design-guidelines/page-layout-patterns.md

```diff
@@ -0,0 +1,145 @@
+
+If there are filters, they are flushed to the left edge of the primary content area.
+
+![contentAction-filter](./img/page-layout-patterns/contentAction-filter.png)
+
+If there is a text filter and an additional filter (e.g.: dropdown or segmented control), the text filter comes first.
+
+![contentAction-txtFilter](./img/page-layout-patterns/contentAction-txtFilter.png)
+
+![contentAction-txtFilter+filter](./img/page-layout-patterns/contentAction-txtFilter+filter.png)
+
+![contentAction-txtFilter+filter+actions](./img/page-layout-patterns/contentAction-txtFilter+filter+actions.png)
+
+#### Examples
+
+##### Standard layout
+
+![standard-example-basic](./img/page-layout-patterns/standard-example-basic.png)
+
+![standard-example-withActions](./img/page-layout-patterns/standard-example-withActions.png)
+
+![standard-example-subheaderSummary](./img/page-layout-patterns/standard-example-subheaderSummary.png)
+
+![standard-example-subheaderPromobanner](./img/page-layout-patterns/standard-example-subheaderPromobanner.png)
+
+![standard-example-tabs](./img/page-layout-patterns/standard-example-tabs.png)
+
+##### Primary content actions
+
+![contentAction-example-withAction](./img/page-layout-patterns/contentAction-example-withAction.png)
+
+![contentAction-example-withTxtFilter](./img/page-layout-patterns/contentAction-example-withTxtFilter.png)
+
+![contentAction-example-withTxtFilter+filter+actions](./img/page-layout-patterns/contentAction-example-withTxtFilter+filter+actions.png)
+
+### Editorial layout
+
+The editorial layout is just like the standard layout. The only differences are that the primary content is constrained to a maximum width, and the primary content doesn't typically have an action area. The maximum width is helpful for maintaining readibility for large blocks of text.
+
+![editorial](./img/page-layout-patterns/editorial.png)
+
+#### Examples
+
+![editorial-example](./img/page-layout-patterns/editorial-example.png)
+
+### Path selection layout
+
+The path selection layout is used when users need to make a selection to determine where they will be taken to complete their task.
+
+The app header bar and app sidebar are hidden when users need to focus on choosing the appropriate path. A simple header may be used to show very basic info (e.g.: page title) and a button to cancel out of the flow.
+
+![pathSelection](./img/page-layout-patterns/pathSelection.png)
+
+#### Examples
+
+![pathSelection-example-one](./img/page-layout-patterns/pathSelection-example-one.png)
+
+![pathSelection-example-two](./img/page-layout-patterns/pathSelection-example-two.png)
+
+### Tabbed form layout
+
+The tabbed form layout is used to display forms with multiple complex sections.
+
+The app header bar and app sidebar are hidden when using this layout to allow users to focus on the complex form. A simple header may be used to show: very basic info (e.g.: page title), a button to cancel out of the form, and a button to submit the form.
```

Comment:
RE simple header... hmmm do we have a more formal name for this? also, i don't think we should say "may" 
Right now, all use cases require this header.

Reply:
button to cancel out should always be secondary action and on left but defer to you to call out or not

Reply:
> RE simple header... hmmm do we have a more formal name for this? also, i don't think we should say "may"
Right now, all use cases require this header.

"simple header" isn't intended to the name of something, it's just a description. I think I'll change this sentence to say:
"A header should be used to show..."